src/Display/Tabs/Tabs.tsx

```diff
@@ -53,13 +53,15 @@ const Tabs: Tabs = ({
             children,
             (data: React.ReactElement<TabPaneProps>, index) => {
               const tabLabel = data.props.label || index;
+              const tabClassName = data.props.className || '';
```

Comment:
@ritikamotwani the `data.props.className` is the `className` passed to `<Tabs.Pane />` right? So it will also apply to panel, could you confirm this? Seems like you want to apply the `className` to tab, so maybe add a `tabClassName` prop to `TabPane`?

Reply:
hmm, I think add a `tabClassName` prop to this `<Tabs />` component is better.

Reply:
what do you think?

Reply:
@tingtingdisco the structure for the Tabs is
```
<Tabs alignment={} className="optional">
<Tabs.Pane tab="name" className="optional-pane"> Body </Tabs.Pane>
<Tabs.Pane tab="Without classname" > Body </Tabs.Pane>
</Tab>
```

So currently the classNames to Tabs do get reflected to the Tabs component, but the classNames to Tabs.Pane are not getting reflected. They shouldn't be reflected to the entire Tab but only to that particular pane.
We'll have to send from here to the Tab.Pane Component just like the tab name is being sent. The thing is the children are getting iterated in this file and then individual ones are getting sent to Tab.Pane component.

Why would I add the tabClassName to Tabs component, they are not related. Tabs has it's own className :thinking:

Reply:
Tabs' className is used to style `TabsContainer`
my concern is when we pass className like this `<Tabs.Pane className="optional-pane">`, I would expect the className is applied to `TabsContent` not the `Tab` where we click on.

Reply:
I think this className is for the Pane only not the content. Because for the content you have to already pass a component inside, you can easily add a className for your content. But currently we cannot set our own className for the Pane we're clicking on.

Reply:
yes, the className is actually being passed to TabsContent in the TabPane component, but the Tabs component is not doing it, it's quite confusing.

So I think we should not make it more confusing by using this className to style `tab`.